CopilotKit/packages/react-textarea/src/components/hovering-toolbar/hovering-toolbar.tsx

```diff
@@ -51,57 +53,82 @@ export const HoveringToolbar = (props: HoveringToolbarProps) => {
 
     const domRange = domSelection.getRangeAt(0);
     const rect = domRange.getBoundingClientRect();
-
+    
     // We use window = (0,0,0,0) as a signal that the selection is not in the original copilot-textarea,
     // but inside the hovering window.
     //
     // in such case, we simply do nothing.
     if (rect.top === 0 && rect.left === 0 && rect.width === 0 && rect.height === 0) {
       return;
     }
-    const minGapFromEdge = 60;
-    const verticalOffsetFromCorner = 35;
-    const horizontalOffsetFromCorner = 15;
```

Comment:
Magic numbers are uncontrolled by the user. And what is "edge" and what is "corner" is pretty unclear. I reset it but left the variables to make future improvements easier.